discovery_engine/lib/src/domain/models/active_data.dart

```diff
@@ -38,12 +41,17 @@ class DocumentWithActiveData with EquatableMixin {
 /// Additional data pertaining to active documents.
 @HiveType(typeId: activeDocumentDataTypeId)
 class ActiveDocumentData with EquatableMixin {
+  @Deprecated('only used in migration')
   @HiveField(0)
   final Embedding smbertEmbedding;
   @HiveField(1)
   final Map<DocumentViewMode, Duration> viewTime;
 
-  ActiveDocumentData(this.smbertEmbedding) : viewTime = {};
+  ActiveDocumentData()
+      :
+        // ignore: deprecated_member_use_from_same_package
+        smbertEmbedding = Embedding(Float32List(0)),
```

Comment:
so we don't use the data stored in hive during migration? could you help me understand why we have a dummy value here?

Reply:
i was under the impression that the generated code for hive on `ActiveDocumentData` uses its own constructor, but i just double checked and it uses this one. i'll revert this and construct empty embeddings in the non-migration related code, basically i just wanted to avoid that.